pkg/k8s/pipelineRun.go

```diff
@@ -135,6 +136,10 @@ func (r *pipelineRun) UpdateState(state api.State) (*api.StateItem, error) {
 	return oldstate, r.updateStatus()
 }
 
+func (r *pipelineRun) GetFullName() string {
```

Comment:
May be implement Stringer instead
https://golang.org/pkg/fmt/#Stringer
func (r *pipelineRun) String() string {
return fmt.Sprintf("%s/%s (%s)",r.apiObj.GetNamespace(),r.apiObj.GetName(),string(r.GetStatus().State))
}

Reply:
I searched for something like this

Reply:
On the other hand. We only want to get the full qualified name. Comparing String() with Java toString() it should also contain (parts of) the parameter values. But even this could be useful.

Reply:
see https://github.com/SAP/stewardci-core/pull/62/commits/343b96a6ab60111eebb064c49d8050afa149940c